spec/requests/admin/site_texts_controller_spec.rb

```diff
@@ -213,11 +213,21 @@
               expected_translations.map do |key, value|
                 overridden =
                   defined?(expected_overridden) ? expected_overridden[key] || false : false
+                interpolation_keys =
+                  (
+                    if defined?(expected_interpolation_keys)
+                      expected_interpolation_keys[key] || []
+                    else
+                      []
+                    end
+                  )
                 {
                   id: "colour.#{key}",
                   value: value,
                   can_revert: overridden,
                   overridden: overridden,
+                  interpolation_keys: interpolation_keys,
+                  has_interpolation_keys: !interpolation_keys.empty?,
```

Comment:
That seems easier to read to me, but up to you:

```suggestion
                  has_interpolation_keys: interpolation_keys.present?,
```